packages/twenty-ui/src/theme/constants/ThemeCommon.ts

```diff
@@ -14,6 +15,20 @@ export const THEME_COMMON = {
   text: TEXT,
   blur: BLUR,
   animation: ANIMATION,
+  snackBar: {
```

Comment:
I'm surprised these are not different between dark and light mode. We could also extract them into their own constant file